src/App.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import SignInForm from './components/Deviseforms/signin';
+import SignUpForm from './components/Deviseforms/signup';
+import Homepage from './components/Homepage/homepage';
+import Splash from './components/Shared/splash';
+import Additempage from './components/Additempage/Additemcomponent';
+import Sidebar from './components/Shared/sidebar';
+import CarDetailsPage from './components/CarDetails/cardetailspage';
+import Reservepage from './components/Reservepage/reservecomponent';
+import MyReservepage from './components/MyReservations/myreservepage';
+import DeletePage from './components/forms/DeletePage';
+
+const App = () => (
```

Comment:
- [ ] Kindly, remove the old commented code found in:

- CarDetails.jsx
- AddItemForm.jsx
- carousel.jsx
- index.css

That way, the repo's cleanliness will be improved. 👍